src/main/java/org/synyx/urlaubsverwaltung/application/web/ApplicationForLeaveViewController.java

```diff
@@ -61,33 +62,53 @@ public ApplicationForLeaveViewController(ApplicationService applicationService,
     @GetMapping("/application")
     public String showWaiting(Model model) {
 
-        List<ApplicationForLeave> applicationsForLeave = getAllRelevantApplicationsForLeave();
+        final List<ApplicationForLeave> applicationsForLeave = getAllRelevantApplicationsForLeave();
         model.addAttribute("applications", applicationsForLeave);
 
+        final List<ApplicationForLeave> applicationsForLeaveCancellationRequests = getAllRelevantApplicationsForLeaveCancellationRequests();
+        model.addAttribute("applications_cancellation_request", applicationsForLeaveCancellationRequests);
+
         return "application/app_list";
     }
 
+    private List<ApplicationForLeave> getAllRelevantApplicationsForLeaveCancellationRequests() {
 
-    private List<ApplicationForLeave> getAllRelevantApplicationsForLeave() {
+        final Person user = personService.getSignedInUser();
+
+        final boolean isBoss = user.hasRole(BOSS);
+        final boolean isOffice = user.hasRole(OFFICE);
+        if (isBoss || isOffice) {
```

Comment:
Why `DepartmentHead` is notified about cancellation request via mail but not allowed to see in summary?

Reply:
I think this is not the case anymore or?